CHANGELOG.md

```diff
@@ -7,6 +7,98 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
+## 2.13.3 (2022-04-14)
+
+This version includes bugfixes for many features of Icinga 2, including fixes for multiple crashes. It also includes a
+number of fixes and improvements for Icinga DB.
+
+### API
+
+* The /v1/config/stages endpoint now immediately rejects parallel config updates instead of accepting and then later
+  failing to verify and activate them. #9328
+
+### Certificates
+
+* The lifetime of newly issued node certificates is reduced from 15 years to 397 days. #9337
+* Compare cluster certificate tickets in constant time. #9333
```

Comment:
Apropos merges: cluster, ...

Reply:
Your suggestion is to merge "API" and "Certificates" into a single "Cluster" section? If so: not sure about that, all changes (except maybe the last) can affect single node icinga2 setups as well.

Reply:
Valid point.